.github/workflows/dependencies.yml

```diff
@@ -33,13 +33,12 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        pip install hatch
         pip install redis==${{ matrix.redis-py-version }}
-        pip install -r requirements.txt -r dev-requirements.txt
-        pip install -e .
 
     - name: Test with pytest
       run: |
-        RUN_SLOW_TESTS_TOO=1 pytest --durations=5
+        RUN_SLOW_TESTS_TOO=1 hatch run test:pytest --durations=5
```

Comment:
Any benefit to using `hatch run test:pytest` instead of calling `pytest` directly? If there's no big benefit I'd rather keep calling `pytest` directly, the less indirection the better.

Reply:
The benefit is that it allows us to define the Python dependencies in the pyproject.toml file in the `[tool.hatch.envs.test]` section and `hatch` creates the virtual environment with those dependencies installed automatically for us, including `rq` itself. This is why we can drop the `pip install` commands above.